### ISSUE TITLE
Large units deploy on playload conveyors

### ISSUE THREAD

**Issue by oinkymoo** [Anuken/Mindustry-Suggestions#2618]
**Describe what you would like changed, and why.**

allow large units to exit reconstructors and payload sources when blocked by payload conveyors. units can already be released onto regular conveyors, and large units can already at least walk on payload conveyors, if they even walk at ground level

**Describe the changes you want to propose. Include possible alternatives.**

see above, i can't think of an alternative

**Comment by itcannotbe** [comment]
Well, it would but then again. Inconsistent behavior is a bad design choice.

TL;DR: No.

**Comment by OnyxFinger** [comment]
> Well, it would but then again. Inconsistent behavior is a bad design choice. TL;DR: No.

inconsistent ? what is inconsistent about this idea ?

**Comment by itcannotbe** [comment]
It's inconsistent because lower tier reconstructors releaae then to the payload conveyor.

If it can't be placed on a payload conveyor it's prevented from exiting.

Also who's dumb enough to put a payload conveyor exiting a T3/T4 reconstructor?

**Comment by OnyxFinger** [comment]
> It's inconsistent because lower tier reconstructors releaae then to the payload conveyor.
> 
> If it can't be placed on a payload conveyor it's prevented from exiting.
> 
> Also who's dumb enough to put a payload conveyor exiting a T3/T4 reconstructor?

huh ? how does that make it inconsistent ?

**Comment by itcannotbe** [comment]
> > It's inconsistent because lower tier reconstructors releaae then to the payload conveyor.
> > If it can't be placed on a payload conveyor it's prevented from exiting.
> > Also who's dumb enough to put a payload conveyor exiting a T3/T4 reconstructor?
> 
> huh ? how does that make it inconsistent ?

The suggestion would make it inconsistent.

**Comment by oinkymoo** [comment]
> > > It's inconsistent because
> > > Reconstructors in general release them to the payload conveyor.
> > > If it can't be placed on a payload conveyor it's prevented from exiting.
> > > Also who's even dumb enough to put a payload conveyor exiting a T3/T4 reconstructor?
> > 
> > 
> > huh ? how does that make it inconsistent ?
> 
> The suggestion would make it inconsistent with the current behavior.

this is incredibly dumb, i'm asking that said current behavior be changed

**Comment by OnyxFinger** [comment]
^^I agree

**Comment by Highfire1** [comment]
(duplicate of #749 :p)

**Comment by oinkymoo** [comment]
> (duplicate of #749 :p)

._. duplicates are allowed now tho and that's an ancient issue anyway

**Comment by itcannotbe** [comment]
It's exactly the same topic, all thing pointed on that one are related to this one.

**Comment by oinkymoo** [comment]
no, the main discussion topic there was payload transport that large units can use, even if the original suggestion was the same
